Order.Management.PR/Program.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Order.Management
+{
+    class Program
+    {
+        // Main entry
+        static void Main(string[] args)
+        {
+            var (customerName, address, dueDate) = CustomerInfoInput();
+
+            var orderedShapes = CustomerOrderInput();
+
+            InvoiceReport(customerName, address, dueDate, orderedShapes);
+
+            CuttingListReport(customerName, address, dueDate, orderedShapes);
+
+            PaintingReport(customerName, address, dueDate, orderedShapes);
+        }
+        
+        // Order Circle Input
+        public static Circle OrderCirclesInput()
+        {
+            Console.Write("\nPlease input the number of Red Circle: ");
+            int redCircle = Convert.ToInt32(userInput());
+            Console.Write("Please input the number of Blue Circle: ");
+            int blueCircle = Convert.ToInt32(userInput());
+            Console.Write("Please input the number of Yellow Circle: ");
+            int yellowCircle = Convert.ToInt32(userInput());
+
+            Circle circle = new Circle(redCircle, blueCircle, yellowCircle);
+            return circle;
+        }
+        
+        // Order Squares Input
+        public static Square OrderSquaresInput()
+        {
+            Console.Write("\nPlease input the number of Red Squares: ");
+            int redSquare = Convert.ToInt32(userInput());
+            Console.Write("Please input the number of Blue Squares: ");
+            int blueSquare = Convert.ToInt32(userInput());
+            Console.Write("Please input the number of Yellow Squares: ");
+            int yellowSquare = Convert.ToInt32(userInput());
+
+            Square square = new Square(redSquare, blueSquare, yellowSquare);
+            return square;
+        }
+
+        // Order Triangles Input
+        public static Triangle OrderTrianglesInput()
+        {
+            Console.Write("\nPlease input the number of Red Triangles: ");
+            int redTriangle = Convert.ToInt32(userInput());
+            Console.Write("Please input the number of Blue Triangles: ");
+            int blueTriangle = Convert.ToInt32(userInput());
+            Console.Write("Please input the number of Yellow Triangles: ");
+            int yellowTriangle = Convert.ToInt32(userInput());
+
+            Triangle triangle = new Triangle(redTriangle, blueTriangle, yellowTriangle);
+            return triangle;
+        }
+
+        // User Console Input
+        public static string userInput()
+        {
+            string input = Console.ReadLine();
+            while (string.IsNullOrEmpty(input))
```

Comment:
The string which contains whitespace only should be invalid as well. Therefore, we should use `string.IsNullOrWhiteSpace` here.